tools/generators/migrate-converged-pkg/schema.json

```diff
@@ -11,7 +11,7 @@
         "$source": "argv",
         "index": 0
       },
-      "pattern": "^[a-zA-Z].*$"
+      "pattern": "^[@a-zA-Z].*$"
```

Comment:
pattern now works. this make it properly pass our package names that may include `@` as first character